activemq-partition/pom.xml

```diff
@@ -92,6 +84,16 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>ch.qos.reload4j</groupId>
+      <artifactId>reload4j</artifactId>
+      <scope>test</scope>
```

Comment:
Personally I would be fine with test scope if this still works when used (by dependants, not within the build tests), but just to note: this is effectively different than what it did in 5.16.3 before (and different than you said you were going to do on the JIRA) since the module wont pass on reload4j like this anymore, wheras log4j was being passed on before due to the zookeeper related bits that brought it in before (but are now excluded via dependencyManagement).